op-node/rollup/types.go

```diff
@@ -287,6 +297,11 @@ func (c *Config) IsFermat(num *big.Int) bool {
 	return isBlockForked(c.Fermat, num)
 }
 
+// IsSnow returns whether the time is either equal to the Snow fork time or greater.
+func (c *Config) IsSnow(time uint64) bool {
+	return isTimestampForked(c.Snow, time)
```

Comment:
This method refers to op-geth, which is the previous Ethereum. The advantage is that the new hard fork is added, this can be easily applied in the future.